src/__tests__/CheckboxGroup.test.tsx

```diff
@@ -172,5 +172,4 @@ describe('CheckboxGroup', () => {
   })
 })
 
-checkStoriesForAxeViolations('CheckboxGroup/fixtures')
-checkStoriesForAxeViolations('CheckboxGroup/examples')
+checkStoriesForAxeViolations('CheckboxGroup.features', '../CheckboxGroup/')
```

Comment:
```suggestion
```

This should be covered by the Playwright tests now 🎉

Reply:
OH what! Okay, so if I add Playwright tests for specific stories I can remove this check?